src/app/content/PrivateHeader.js

```diff
@@ -37,6 +38,11 @@ const PrivateHeader = ({ search, onSearch, location }) => {
                             {c('Title').t`Contacts`}
                         </Link>
                     </li>
+                    {isFree ? (
```

Comment:
`isFree` means free VPN and Mail. Here you need to use the `hasPaidMail` coming from the same hook.